lib/fog/azurerm/config.rb

```diff
@@ -1,2 +1,5 @@
 AZURE_RESOURCE = 'https://management.azure.com'.freeze
 DEFAULT_ADDRESS_PREFIXES = ['10.2.0.0/16'].freeze
+SUBNET = 'Subnet'.freeze
```

Comment:
Faiz and Umair has suggested this name after code review. 
As discussed with you, we will not incorporate this change.